recipe_scrapers/farmhousedelivery.py

```diff
@@ -0,0 +1,94 @@
+import re
+
+from bs4 import Tag
+
+from ._abstract import AbstractScraper
+from ._utils import normalize_string
+
+
+class FarmhouseDelivery(AbstractScraper):
+    @classmethod
+    def host(self, domain="com"):
+        return f"recipes.farmhousedelivery.{domain}"
+
+    def title(self):
+        return self.soup.find("h1", {"class": "entry-title"}).get_text(strip=True)
+
+    def ingredients(self):
+        # Style 1
+        ingredients_marker = self.soup.find("p", text=re.compile(r"Ingredients:"))
+        if ingredients_marker is not None:
+            ingredients_marker_siblings = ingredients_marker.next_siblings
+            for ingredients_marker_sibling in ingredients_marker_siblings:
+                if (
+                    isinstance(ingredients_marker_sibling, Tag)
+                    and ingredients_marker_sibling.name == "ul"
+                ):
+                    ingredients = ingredients_marker_sibling.findAll("li")
+                    return [
+                        normalize_string(ingredient.get_text(strip=True))
+                        for ingredient in ingredients
+                    ]
+
+        # Style 2
+        ingredients_marker = self.soup.find("p", text=re.compile(r"Ingredients"))
+        if ingredients_marker is not None:
+            ingredients = []
+            ingredients_marker_siblings = ingredients_marker.next_siblings
+            for ingredients_marker_sibling in ingredients_marker_siblings:
+                if (
+                    isinstance(ingredients_marker_sibling, Tag)
+                    and ingredients_marker_sibling.name == "p"
+                ):
+                    if ingredients_marker_sibling.get_text() == "Instructions":
+                        break
+                    else:
+                        ingredients.append(ingredients_marker_sibling)
+            return [
+                normalize_string(ingredient.get_text()) for ingredient in ingredients
+            ]
```

Comment:
Instead of looping through the `ingredients` list could we just normalize and call `get_text()` right on `ingriedients_marker_sibling`? That way we get everything done in one loop.

Reply:
Thanks for the recommendation.  Code updated.